backend/src/main/java/com/hackner/musicompass/controller/ArtistController.java

```diff
@@ -0,0 +1,26 @@
+package com.hackner.musicompass.controller;
+
+import com.hackner.musicompass.model.Artist;
+import com.hackner.musicompass.service.ArtistService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/artist")
```

Comment:
Coaches would say it would be better to use `'api/artist'` - also you do not need to include `/` at the beginning

Reply:
thx. done.